.env.debug

```diff
@@ -0,0 +1 @@
+DEBUG_LEVEL=VERBOSE
```

Comment:
Should probably be `LOG_LEVEL` because otherwise you're inevitably gonna end up with `DEBUG_LEVEL=DEBUG`, which is quite unhelpful

Reply:
Changed this